### PR TITLE
Passive Buzzer Documentation

### PR DESCRIPTION
**Pull request by adelgadoj3**
Created passive buzzer documentation, added circuit diagram, and added link to index documentation

### CODE REVIEW

**Review by ruthvikm — CHANGES_REQUESTED** [review]
- [ ] Add example controller at the end.
- [ ] Add Passive Buzzer Javadoc in the functionality section. Refer to touch switch documentation.

**Review by joefol — APPROVED** [review]
Looks good to me!

**Comment by adelgadoj3** [comment]
> * [x]  Add example controller at the end.
> * [x]  Add Passive Buzzer Javadoc in the functionality section. Refer to touch switch documentation.

Should I incorporate the schematic diagram and/or the buzzer diagram as well?

image


image

File: pi4micronaut-utils/src/docs/asciidoc/index.adoc [hunk 1]
```diff
@@ -77,4 +77,6 @@ include::components/outputComponents/rgbLed.adoc[]
 
 include::components/outputComponents/lcd1602.adoc[]
 
+include :: components/outputComponents/passivebuzzer.adoc[]
```
Comment (ruthvikm) [inline pi4micronaut-utils/src/docs/asciidoc/index.adoc:80]: Remove space here -  include::components....

**Review by ruthvikm — CHANGES_REQUESTED** [review]
To add example controller, you need to add //tag::ex[] in passivebuzzerController class

**Comment by ruthvikm** [comment]
Schematic diagram will be useful

File: pi4micronaut-utils/src/docs/asciidoc/components/outputComponents/passivebuzzer.adoc [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+      shutdown: 0
+----
+
+===== Example Controller
+
+====== This controller uses the Passive Buzzer to emit sound once prompted by the commands
+
+[source, java]
+----
+include::../../../../../../components/src/main/java/com/opensourcewithslu/components/controllers/PassiveBuzzerController.java[tag=ex]
```
Comment (SwindleA) [inline pi4micronaut-utils/src/docs/asciidoc/components/outputComponents/passivebuzzer.adoc:90]: The example controller is not showing up for me.

**Review by SwindleA — CHANGES_REQUESTED** [review]
other than that one possible issue. It looks good.

File: pi4micronaut-utils/src/docs/asciidoc/components/outputComponents/passivebuzzer.adoc (hunk 2 above)
Comment (ruthvikm) [reply pi4micronaut-utils/src/docs/asciidoc/components/outputComponents/passivebuzzer.adoc:90]: tag was not added to the controller

File: pi4micronaut-utils/src/docs/asciidoc/components/outputComponents/passivebuzzer.adoc [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+
+
+
+===== Troubleshooting
+- Passive Buzzer not creating sound:
+    * Double check power source
+    * Verify that 1K resistor is used
+    * Verify all wires are in appropriate slots.
+    * When using a file, verify that the file itself is scp'd to the raspberrypi (see PassiveBuzzerController for full information).
+
```
Comment (ruthvikm) [inline pi4micronaut-utils/src/docs/asciidoc/components/outputComponents/passivebuzzer.adoc:70]: Add constructors and methods senction with passiveBuzzer javadoc link.
https://oss-slu.github.io/Pi4Micronaut/javadoc/com/opensourcewithslu/outputdevices/PassiveBuzzerHelper.html

**Review by ruthvikm — CHANGES_REQUESTED** [review]
Resolve conflicts

File: pi4micronaut-utils/src/docs/asciidoc/components/outputComponents/passivebuzzer.adoc [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+* Connect a jumper wire from GPIO 17 to the 1KΩ resistor
+* Connect jumper wire from positive side to positive end of Passive Buzzer.
+* Connect jumper wire from negative end of Passive Buzzer to emitter pin of S8550 PNP Transistor.
+* Connect jumper wire from negative side to collector pin of S8550 PNP Transistor.
+
+
+===== Circuit Diagram
+Schematic Diagram
+
+image::buzzer-SD.png[]
```
Comment (ruthvikm) [inline pi4micronaut-utils/src/docs/asciidoc/components/outputComponents/passivebuzzer.adoc:50]: this image was not uploaded

**Review by ruthvikm — CHANGES_REQUESTED** [review]

File: pi4micronaut-utils/src/docs/asciidoc/components/outputComponents/passivebuzzer.adoc (hunk 4 above)
Comment (adelgadoj3) [reply pi4micronaut-utils/src/docs/asciidoc/components/outputComponents/passivebuzzer.adoc:50]: I added the image in the active buzzer documentation issue.

**Review by ruthvikm — APPROVED** [review]

**Review by ruthvikm — CHANGES_REQUESTED** [review]
Resolve conflicts

**Review by ruthvikm — APPROVED** [review]

**Pull request merged**